src/components/GeolocationTracking/GeolocationLogsExporter.jsx

```diff
@@ -10,14 +9,9 @@ import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
 import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
 
 const GeolocationLogsExporter = () => {
-  const { isGeolocationTrackingAvailable } = useGeolocationTracking()
   const { t } = useI18n()
   const webviewIntent = useWebviewIntent()
 
-  if (!isGeolocationTrackingAvailable) {
-    return null
-  }
```

Comment:
Is it safe to remove this ?

Reply:
Oh you added the check before.

Reply:
Oui, la condition est gérée dans Settings, comme pour les autre composants qui sont également conditionnés, ça normalise un peu.